applications/callflow/src/module/cf_branch_property.erl

```diff
@@ -0,0 +1,92 @@
+%%%-------------------------------------------------------------------
+%%% @copyright (C) 2013-2016, 2600Hz INC
+%%% @doc
+%%%
+%%% "data":{
+%%%   "action":"branch", % the only one action supported currently
+%%%   "scope": "[device|user|account]",
+%%%   "property": "whatever_field_you_want"
+%%% }
+%%%
+%%% Branch to child whose name pointed in callflow's data section
+%%%
+%%% @end
+%%% @contributors
+%%%   SIPLABS LLC (Maksim Krzhemenevskiy)
+%%%-------------------------------------------------------------------
+-module(cf_branch_property).
+
+-include("../callflow.hrl").
+
+%% API
+-export([handle/2]).
+
+-spec handle(wh_json:object(), whapps_call:call()) -> 'ok'.
+handle(Data, Call) ->
+    Scope = wh_json:get_binary_value(<<"scope">>, Data),
+    PointedField = wh_json:get_binary_value(<<"property">>, Data),
```

Comment:
For now - yes. This could be expanded in future versions (we need to invent some syntax for that)